_pkgdown.yml

```
@@ -96,95 +96,151 @@ news:
     href: https://shiny.posit.co/blog/posts/announcing-new-r-shiny-ui-components/
 
 reference:
-- title: Create a dashboard
-  contents:
+- title: Page Layouts
+  desc: >
+    Page layouts are the basic building blocks of any UI. bslib pages always
+    include Bootstrap and are can be themed with bslib's theming capabilities.
```

Maybe we could also hyperlink to the theming section here?

```suggestion
    include Bootstrap and can be themed with bslib's theming capabilities.
```

---

```suggestion
    include Bootstrap and can be themed with bslib's [theming capabilities](#theming).
```